gc/gc-base/src/test/java/org/projectnessie/gc/base/TestJerseyRestGcInMemory.java

```diff
@@ -23,11 +23,17 @@
 import org.projectnessie.jaxrs.ext.NessieJaxRsExtension;
 import org.projectnessie.jaxrs.ext.NessieUri;
 import org.projectnessie.versioned.persist.adapter.DatabaseAdapter;
+import org.projectnessie.versioned.persist.inmem.InmemoryDatabaseAdapterFactory;
+import org.projectnessie.versioned.persist.inmem.InmemoryTestConnectionProviderSource;
 import org.projectnessie.versioned.persist.tests.extension.DatabaseAdapterExtension;
 import org.projectnessie.versioned.persist.tests.extension.NessieDbAdapter;
+import org.projectnessie.versioned.persist.tests.extension.NessieDbAdapterName;
+import org.projectnessie.versioned.persist.tests.extension.NessieExternalDatabase;
 
+@NessieDbAdapterName(InmemoryDatabaseAdapterFactory.NAME)
+@NessieExternalDatabase(InmemoryTestConnectionProviderSource.class)
 @ExtendWith(DatabaseAdapterExtension.class)
-abstract class AbstractTestJerseyRestGC extends AbstractRestGCTest {
+class TestJerseyRestGcInMemory extends AbstractRestGCTest {
```

Comment:
Also combined `TestJerseyRestGcInMemory` and `AbstractTestJerseyRestGC` into just `TestJerseyRestGcInMemory`